CONTRIBUTING.md

```diff
@@ -179,16 +179,57 @@ and instead you should
 ```python
 
 def function_with_many_args(
-    argument1,
-    argument2,
-    argument3,
-    argument4,
-    argument5,
-    argument6,
-    argument7):
+                            argument1,
+                            argument2,
+                            argument3,
+                            argument4,
+                            argument5,
+                            argument6,
+                            argument7,
+):
     pass
 ```
 
+**Functions with type annotations**
+
+When using type annotations the function should be just like in the above section but also include
+the types of the arguments and the return type.
+
+```python
+def a(
+        b: B,
```

Comment:
Do we need the 8 spaces here, 4 would be enough IMO

Reply:
yes we do. This is the style we have been following so far.